site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/support/components/FileTable.js

```diff
@@ -8,9 +8,9 @@ import {
 import { humanReadableBytes } from "react-invenio-deposit";
 
 
-const FileTable = ({ filesList, handleDelete, decimalSizeDisplay }) => {
+const FileTable = ({ filesList, handleDelete, decimalSizeDisplay, negative }) => {
     return (
-        <Table>
+        <Table className={negative ? "negative" : ""}>
```

Comment:
Minor: I think that the [Table cmp](https://react.semantic-ui.com/collections/table/#states-positive-negative) from SUIR has a negative prop that could be used in this case and avoid the custom css, WDYT?
```suggestion
        <Table>
```
Down you could use:
```diff
- <Table.Row key={file.name}>
+ <Table.Row key={file.name} negative={negative}>
```

Reply:
Oh I didn't know! Will check it out, thanks!

Reply:
werent we using the class negative instead the props because of UX related color theme overrides ? or those were different components, not table?